redis/connection.py

```diff
@@ -548,9 +550,11 @@ def __init__(
         self.host = host
         self.port = int(port)
         self.db = db
-        self.username = username
         self.client_name = client_name
-        self.password = password
```

Comment:
Removing these attributes could be a breaking change for some apps.

Reply:
added back